pkg/blockdeviceclaim/v1alpha1/blockdeviceclaim.go

```diff
@@ -104,6 +104,19 @@ func (bdc *BlockDeviceClaim) IsStatus(status string) bool {
 }
 
 // Len returns the length og BlockDeviceClaimList.
-func (l *BlockDeviceClaimList) Len() int {
-	return len(l.ObjectList.Items)
+func (bdcl *BlockDeviceClaimList) Len() int {
+	return len(bdcl.ObjectList.Items)
+}
+
+// GetBDList returns map of node name and corresponding block devices to that
+// node from block device claim list
+func (bdcl *BlockDeviceClaimList) GetBDList() map[string][]string {
```

Comment:
Can we rename this to `GetBlockDeviceNamesByNode`

Reply:
Done